solvebio/client.py

```diff
@@ -103,26 +104,39 @@ def __init__(self, host=None, token=None, token_type='Token',
         }
         self.set_user_agent()
 
+        solvebio_retry_all = os.environ.get('SOLVEBIO_RETRY_ALL')
```

Comment:
I would just wrap this in a `bool()` rather than require `1`

Reply:
👍